src/general/base_stage/StageBase.cs

```diff
@@ -38,6 +38,11 @@ public abstract class StageBase : NodeWithInput, IStageBase, IGodotEarlyNodeReso
 
     private bool transitionFinished;
 
+    /// <summary>
+    ///   Used to control how often updated light level data is read from the day/night cycle.
+    /// </summary>
+    private float elapsedSinceLightLevelUpdate;
```

Comment:
I didn't make this savable as I think the value is just too insignificant to be saved, plus I think we've been making similar fields as not for saving lately.

Reply:
Yeah, this is so inconsequential that I wouldn't save this. Though maybe this should default to 1 to ensure that an immediate update happens?

Reply:
If you don't mind, I'll commit this change and approve this PR now. I just playtested and I think this solved all the issues I had.

Reply:
Sure.